README.md

```diff
@@ -122,3 +122,23 @@ The two models selected were ViLT and LXMERT. The goal is to compare these model
 ![LXMERT Visualization](https://raw.githubusercontent.com/hila-chefer/Transformer-MM-Explainability/main/LXMERT.PNG)
 *Source: Generic Attention-model Explainability for Interpreting Bi-Modal and Encoder-Decoder Transformers (2021)*
 
+## LXMERT Model Training and Fine-tuning Process
```

Comment:
Can we add this heading (or a more generic heading for training) to the table of contents? This will make it easier for someone to click to from the beginning of the README

Reply:
@marcoperic